Sources/MapboxCoreNavigation/Distance/GantryDistanceInfo.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import MapboxNavigationNative
+
+/**
+ * Contains information about distance to the road object represented as gantry.
+ */
+public struct GantryDistanceInfo {
+
+    /** Distance to the gantry object */
+    public let distance: CLLocationDistance
+
+    public init(distance: CLLocationDistance) {
```

Comment:
Several public initializers added in this PR all need documentation comments.

Reply:
Yes, I've added public initializers to most of the EH structs as a response to 1Tap requests to improve testing capabilities. We've discussed this approach with Android team, and they added builders to an internal module so that additions to APIs won't break APIs.
Do we want to investigate other approaches not to make any additions a breaking change?

Reply:
We wouldn’t use builders in this Swift API, but in general the way we’d handle future additions would be to make them optional: either optional initializer arguments with `nil` default values or optional properties with `nil` initial values.

Reply:
Documentation was added as part of #2991.